report/.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
added a new `eslintrc` for `report/` to make it `env` `browser` and remove the ad hoc `/* global document window */`  and `/* eslint-env browser */`s sprinkled throughout some of the files.

Also, turns out eslintrc files [automatically extend](https://eslint.org/docs/user-guide/configuring/configuration-files#cascading-and-hierarchy) any other eslintrc files up through to the root directory, though I added a `root: true` in the root file just to make it clear